Assets/Scripts/Audio/AudioSpectrumDrawer.cs

```diff
@@ -0,0 +1,90 @@
+using ProefExamen.Audio.WaveFormDrawer;
+using UnityEngine;
+
+namespace ProefExamen.Audio.SpectrumDrawer
+{
+    public class AudioSpectrumDrawer : MonoBehaviour
+    {
+        [SerializeField]
+        private int _audioSamples = 512;
+
+        [SerializeField]
+        private float _lerpAmount = 20;
+
+        [SerializeField]
+        private float _maxScale = 1200;
+
+        [SerializeField]
+        private Vector3 _offsetPosition = Vector3.zero;
+
+        [Space]
+        [SerializeField]
+        private GameObject _imagePrefab;
+
+        private GameObject[] _visualizers;
+        private AudioWaveformDrawer waveformDrawer;
```

Comment:
```suggestion
        private AudioWaveformDrawer _waveformDrawer;
```
Serialized or regular private always needs an _.